src/pages/understand-stacks/running-api-node.md

```diff
@@ -0,0 +1,300 @@
+
+To verify the database is ready:
+
+1. Connect to the Postgres instance with the command `psql -h localhost -U postgres`. Use the password from the
+   `POSTGRES_PASSWORD` environment variable you set when running the container.
+2. List current databases with the command `\l`
+3. Disconnect from the database with the command `\q`
+
+To verify the `stacks-blockchain` tip height is progressing use the following command:
+
+```sh
+curl -sL localhost:20443/v2/info | jq
+```
+
+If the instance is running you should recieve terminal output similar to the following:
+
+```json
+{
+  "peer_version": 402653184,
+  "pox_consensus": "89d752034e73ed10d3b97e6bcf3cff53367b4166",
+  "burn_block_height": 666143,
+  "stable_pox_consensus": "707f26d9d0d1b4c62881a093c99f9232bc74e744",
+  "stable_burn_block_height": 666136,
+  "server_version": "stacks-node 2.0.11.1.0-rc1 (master:67dccdf, release build, linux [x86_64])",
+  "network_id": 1,
+  "parent_network_id": 3652501241,
+  "stacks_tip_height": 61,
+  "stacks_tip": "e08b2fe3dce36fd6d015c2a839c8eb0885cbe29119c1e2a581f75bc5814bce6f",
+  "stacks_tip_consensus_hash": "ad9f4cb6155a5b4f5dcb719d0f6bee043038bc63",
+  "genesis_chainstate_hash": "74237aa39aa50a83de11a4f53e9d3bb7d43461d1de9873f402e5453ae60bc59b",
+  "unanchored_tip": "74d172df8f8934b468c5b0af2efdefe938e9848772d69bcaeffcfe1d6c6ef041",
+  "unanchored_seq": 0,
+  "exit_at_block_height": null
+}
+```
+
+Verify the `stacks-blockchain-api` is receiving data from the `stacks-blockchain` with the following command:
+
+```sh
+curl -sL localhost:3999/v2/info | jq
+```
+
+If the instance is configured correctly, you should recieve terminal output similar to the following:
+
+```json
+{
+  "peer_version": 402653184,
+  "pox_consensus": "e472cadc17dcf3bc1afafc6aa595899e55f25b72",
+  "burn_block_height": 666144,
+  "stable_pox_consensus": "6a6fb0aa75a8acd4919f56c9c4c81ce5bc42cac1",
+  "stable_burn_block_height": 666137,
+  "server_version": "stacks-node 2.0.11.1.0-rc1 (master:67dccdf, release build, linux [x86_64])",
+  "network_id": 1,
+  "parent_network_id": 3652501241,
+  "stacks_tip_height": 61,
+  "stacks_tip": "e08b2fe3dce36fd6d015c2a839c8eb0885cbe29119c1e2a581f75bc5814bce6f",
+  "stacks_tip_consensus_hash": "ad9f4cb6155a5b4f5dcb719d0f6bee043038bc63",
+  "genesis_chainstate_hash": "74237aa39aa50a83de11a4f53e9d3bb7d43461d1de9873f402e5453ae60bc59b",
+  "unanchored_tip": "74d172df8f8934b468c5b0af2efdefe938e9848772d69bcaeffcfe1d6c6ef041",
+  "unanchored_seq": 0,
+  "exit_at_block_height": null
+}
+```
+
+## Stopping the API node
+
+As discussed previously, if you want to bring down your API node, you must stop the services in the reverse order that
+you started them. Performing the shutdown in this order ensures that you don't lose any data while shutting down
+the node.
+
+Use the following commands to stop the local API node:
+
+```sh
+docker stop stacks-blockchain
+docker stop stacks-blockchain-api
+docker stop postgres
+```
+
+## Additional reading
```

Comment:
Some missing links here

Reply:
These are also remote links, defined in lines 298 and 299